test/integration/images/sti-fake-onbuild/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM sti_test/sti-fake
+
+ONBUILD RUN touch /sti-fake/src/onbuild
+
+# The ONBUILD strategy only works with the application source dir so we need
+# to manually specify to copy to another location.
+#
+# This is a little hack-ish given that we know our assemble script requires files to be in /tmp/src
+# we will copy there, and also set our WORKDIR to be the same location so it has access to the scripts
+ONBUILD COPY . /tmp/src/
```

Comment:
By default all onbuild images copy current directory when building (eg. [python:onbuild](https://github.com/docker-library/python/blob/7663560df7547e69d13b1b548675502f4e0917d1/3.4/onbuild/Dockerfile#L9)). Aside from the application sources current directory contains also `assemble` and `run` scripts, we need to make sure they are copied into the image, because if you have a look at generated `Dockerfile` you'll notice that right after `FROM` line we put `RUN sh assemble` [line](https://github.com/openshift/source-to-image/blob/master/pkg/build/strategies/onbuild/onbuild.go#L168-L170). The reason why we are copying it to `/tmp/src/` is we set WORKDIR to be `/tmp/src/` in the next line. I hope that helps :smiley: